syntax = "proto3";

package unifmu_handshake;

// https://developers.google.com/protocol-buffers/docs/reference/java-generated#package
option java_package = "";
option java_outer_classname = "UnifmuHandshake";

enum HandshakeStatus {
    STATUS_UNSPECIFIED = 0;
    OK = 1;
    ERROR = 2;
}

message HandshakeRequest {}

message HandshakeReply {
    HandshakeStatus status = 1;
}